view/template/trial.hbs

```diff
@@ -180,4 +180,5 @@
 <script>
     window.data = JSON.parse('{{{ graphData }}}');
 </script>
+<script type="text/javascript" src="http://canvasjs.com/assets/script/canvasjs.min.js" defer></script>
```

Comment:
this is unneeded, chart js is already loaded on the page from `/view/layout/default.hbs`.